app/src/main/java/com/globant/onboardingcalculatorwithkotlin/mvp/view/CalculatorView.kt

```diff
@@ -0,0 +1,36 @@
+package com.globant.onboardingcalculatorwithkotlin.mvp.view
+
+import android.app.Activity
+import android.widget.Toast
+import com.globant.onboardingcalculatorwithkotlin.R
+import com.globant.onboardingcalculatorwithkotlin.mvp.contracts.CalculatorContracts
+import com.globant.onboardingcalculatorwithkotlin.utils.Constants.NUMBER_ZERO
+import kotlinx.android.synthetic.main.activity_main.visor
+
+class CalculatorView(activity: Activity) : ActivityView(activity), CalculatorContracts.View {
+
+    override fun refreshVisor(value: String) {
+        activity?.visor?.text = value
```

Comment:
put the views in private val variables and use the variables

Reply:
Done!